tests/python/gaia-ui-tests/gaiatest/tests/functional/bzlite/test_fillbug.py

```diff
@@ -0,0 +1,33 @@
+from gaiatest import GaiaTestCase
+from gaiatest.apps.homescreen.regions.confirm_install import ConfirmInstall
+import time
+from gaiatest.apps.bzlite.app import BugzillaLiteStage
+
+
+class TestFillBug(GaiaTestCase):
+    def setUp(self):
+        GaiaTestCase.setUp(self)
+        self.connect_to_local_area_network()
+        self.test_app = {
+            'name': 'Bugzilla Lite Stage',
+            'url': 'http://bzlite-staging.herokuapp.com/manifest.webapp'}
+        self.logger.info('Test data: %s' % self.test_app)
+        self.marionette.execute_script(
+            'navigator.mozApps.install("%s")' % self.test_app['url'])
+        confirm_install = ConfirmInstall(self.marionette)
+        confirm_install.tap_confirm()
+        self.push_resource('IMG_0001.jpg')
```

Comment:
@mwargers here we are installing bugzilla Lite stagging version and name it Bugzilla Lite Stage.
We need to use the stage version because we are filling some bugs for test and we have to use https://bugzilla-dev.allizom.org/ which bugzilla lite Stage is pointed on, I hope I explained it well, cc @JohanLorenzo